go/vt/dbconfigs/dbconfigs.go

```diff
@@ -394,6 +394,12 @@ func (dbcfgs *DBConfigs) getParams(userKey string, dbc *DBConfigs) (*UserConfig,
 	return uc, cp
 }
 
+// SetDbParams sets the dba and app params
+func (dbcfgs *DBConfigs) SetDbParams(params mysql.ConnParams) {
```

Comment:
this feels too specific to the usage. If it is going in here, maybe it should be
```
SetDbParams(dbaParams, appParams mysql.ConnParams)
```

Reply:
I did not find the method to set those params and those params are being used by the engine internally to connect the binlog server, that is why I added this.